### PR TITLE
Created employee class w/ getters and constructor

### PR DESCRIPTION
**Pull request by AndrewC243**

### CODE REVIEW

**Review by mario-gonzalez-h — DISMISSED** [review]
Leaving us in the dust I see XD

File: src/main/java/org/example/Employee.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package org.example;
+
+public class Employee {
+    private int emplyoeeId;
```
Comment (beatsnmelody) [inline src/main/java/org/example/Employee.java]: You spelled employee wrong here, which might lead to confusion later! Other than that, looks good!

Comment (AndrewC243) [reply src/main/java/org/example/Employee.java]: Good catch; fixed!

**Review by beatsnmelody — APPROVED** [review]
Looks good now!

**Pull request merged**